### DISCUSSION TITLE
Duplicate or common template or central control tiles

### DISCUSSION THREAD

**Discussion by nomandera** [linuxserver/Heimdall#691] [Ideas]
With the advent of multi user support I find my self having to add manually the same common tiles to many users.

It perhaps makes sense to consider:

-  Duplicate a user feature
- A default template for new users
- A tick box in the admin account that allows simple choosing of which tiles get pushed out to which user (or removed)

I like all these methods although as the admin I think the option to be able to see in one view what the system is being used for and control (configure, add or remove) tiles would have extra benefits.

If nothing else it would lay the foundation for RO users i.e. kids

**Comment by Wdavery** [comment]
This is big, having to manually duplicate a dozen/two dozen etc tiles becomes tedious very quickly. Especially when some are custom tiles with custom icons.

**Comment by KodeStar** [comment]
There is kind of a hack in 2.2.1, if you create a single item then open up the database in DB Browser for SQlite, go to Browse data tab, select the `items` table from the dropdown, find your item, scroll to the right, double click on the user_id, change it to 0, click apply, then click Write changes and the item will be visible to all users. Here is a video to make it a bit clearer - https://drive.google.com/file/d/1gJ3xlr2HKPt9USbkIWoOdGWqVSQ0KIcg/view?usp=sharing

**Comment by bob454522** [comment]
i would love anyway to clone or duplicate an existing tile- 
there are so many times i need to change just one or two characters of an existing tile (like same service, but on another IP) 
  and have to end up copying every field, then paste each into the new tile (and make the small change) , then save.
(i use a multiple clipboards app, as otherwise i would have to use notepad or similar to temporarily hold the contents while i made a new tile.
with a "duplicate/clone tile" option,  the above could change into a very easy 3 step process!
thanks,  i do love Heimdall!
